gno.land/pkg/gnoweb/handler.go

```diff
@@ -196,17 +183,13 @@ func (h *WebHandler) GetRealmView(gnourl *weburl.GnoURL, indexData *components.I
 	})
 }
 
-func (h *WebHandler) GetHelpView(gnourl *weburl.GnoURL, indexData *components.IndexData) (int, *components.View) {
+func (h *WebHandler) GetHelpView(gnourl *weburl.GnoURL) (int, *components.View) {
 	fsigs, err := h.Client.Functions(gnourl.Path)
 	if err != nil {
 		h.Logger.Error("unable to fetch path functions", "error", err)
 		return GetClientErrorStatusPage(gnourl, err)
 	}
 
-	// HTML Head metadata
```

Comment:
@moul Should we keep these ones at least (default ones for source, actions, dir)? There is no UGC in those one (so no UGC risk)

Reply:
Sure, we can keep whatever you want. I tried to manage conflicts, but I'm not very familiar with this feature. I offer you two choices: continue working on my pull request to improve it, or we can merge this revert and you can focus on SEO optimizations later.